BaselineOfPTerm/BaselineOfPTerm.class.st

```diff
@@ -6,15 +6,26 @@ Class {
 
 { #category : #baselines }
 BaselineOfPTerm >> baseline: spec [
+
 	<baseline>
-spec for: #common do: [ 
-	"spec preLoadDoIt: #preload."
-	spec
-		package: 'PTerm-Core' with:  [ spec requires: #('PTerm-Protocol') ];
-		package: 'PTerm-Test';
-		package: 'PTerm-Protocol';
-		package: 'PTerm-Fonts';
-		package: 'PTerm-UI';
-		group: 'default' with: #('PTerm-Core' 'PTerm-Test' 'PTerm-Fonts' 'PTerm-UI')
+	spec for: #common do: [
+		spec
+			package: 'PTerm-Core'
+			with: [ spec requires: #( 'PTerm-Protocol' ) ];
+			package: 'PTerm-Test';
+			package: 'PTerm-Protocol';
+			package: 'PTerm-Fonts';
+			package: 'PTerm-UI';
+			group: 'default' with: #( 'PTerm-Core' 'PTerm-Test' 'PTerm-Fonts' 'PTerm-UI' ) ].
+
+	spec for: #'pharo13.x' do: [
```

Comment:
This spec may be reusable for any future version of Pharo,  not just Pharo 13. It is better if we can generalize it (something like >= pharo13)